content/en/docs/setup/install/operator/index.md

```diff
@@ -75,25 +73,214 @@ accessible to `istioctl` by using this command:
+1. [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+1. [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
+1. [Pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [Service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/)
+1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
+1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+
+All of these K8s settings use the K8s API definitions, so [K8s documentation](https://kubernetes.io/docs/concepts/) can be used for reference.
+
+### Configure the feature or component
+
+After you indentify the name of the feature or component from the previous list, you can use the API to set the values using a configuration overlay file.
+
+For example, use this overlay file ([samples/pilot-k8s.yaml](https://github.com/istio/operator/blob/release-1.3/samples/pilot-k8s.yaml))
+to adjust the resources and HPA scaling settings for Pilot, which is a part of the TrafficManagement feature:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  trafficManagement:
+    components:
+      pilot:
+        k8s:
+          resources:
+            requests:
+              cpu: 1000m # override from default 500m
+              memory: 4096Mi # ... default 2048Mi
+          hpaSpec:
+            maxReplicas: 10 # ... default 5
+            minReplicas: 2  # ... default 1
+{{< /text >}}
+
+## Customize Istio settings using the Helm API
+
+The [Helm API](/docs/reference/config/installation-options/) is available as part of the Operator API
+through the `values` field in IstioControlPlane
+(for [global settings](/docs/reference/config/installation-options/#global-options))
+and per-component `values` fields for each Istio component.
+
+For example, the following YAML file configures some global and Pilot settings through the Helm API:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  trafficManagement:
+    components:
+      pilot:
+        values:
+          traceSampling: 0.1 # override from 1.0
+
+  # global Helm settings
+  values:
+    monitoringPort: 15050
+{{< /text >}}
+
+Some parameters will temporarily exist in both the Helm and IstioControlPlane APIs - for example, K8s resources,
+namespaces and enablement. However, the Istio community recommends using the IstioControlPlane API as it is more
+consistent, is validated, and follows the graduation process for APIs.
```

Comment:
remove ` is`

Reply:
changes `follows` to `uses`

Reply:
I don't agree with this change:
- "is validated" seems more grammatically correct in this case. 
- "follows" seems to me to better capture the fact that this is a multi-stage process.

Reply:
right.  I think I left out one comment when i started that 👍 

```
`IstioControlPlane` API is consistent, validated, and follows the [Istio API](link to community process for this) graduation process.
```

This does us the good deed of casting the operator in a positive light without making a direct comparison to our old work.

Reply:
I think this comment was missed.